index.bs

```diff
@@ -361,13 +361,13 @@ one line except when a line break is definitely necessary.</p>
  <pre>
  WEBVTT
 
- 00:00:00.000 --> 00:00:04.000 position:10%,start align:left size:35%
+ 00:00:00.000 --> 00:00:04.000 position:10%,left align:left size:35%
```

Comment:
I think of it as [line-left](https://drafts.csswg.org/css-writing-modes-3/#line-left), mirroring the behavior of `align:left`. "start" implies different sides for LTR and RTL base direction, which is not the case here.